docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **3**                  | MAY          | The CA MAY include additional `PolicyInformation` values that meet the following profile: |
+| \ \ \ \ `policyIdentifier` | MUST         | An identifier documented by the CA in its Certificate Policy and/or Certification Practice Statement. |
+| \ \ \ \ `policyQualifiers` | SHOULD NOT   | See below for restrictions on `policyQualifiers` |
+| \ \ **4**                  | MUST NOT     | The CA MUST NOT include any additional `PolicyInformation` values that do not meet the above profile. |
+
+If the `policyQualifiers` is permitted and present within a `PolicyInformation` field, it MUST be formatted as follows:
+
+| __Qualifier ID__                     | __Presence__ | __Field Type__ |  __Contents__ |
+| ---                                  | -            | -              | -----         |
+| `id-qt-cps` (OID: 1.3.6.1.5.5.7.2.1) | MAY          | `IA5String`    | The HTTP URL for the Issuing CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the Issuing CA. |
+| Any other qualifier                  | MUST NOT     | -              | -             |
+
+##### 7.1.2.8.6 Extended Key Usage - Non-TLS CA
+
+The Issuing CA MUST verify that the Subordinate CA Certificate is authorized to issue certificates for each included extended key usage purpose. Multiple, independent key purposes (e.g. `id-kp-timeStamping` and `id-kp-codeSigning`) SHOULD NOT be present.
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST NOT     |
+| `id-kp-clientAuth`      | 1.3.6.1.5.5.7.3.2 | MAY          |
+| `id-kp-codeSigning`     | 1.3.6.1.5.5.7.3.3 | MAY          |
+| `id-kp-emailProtection` | 1.3.6.1.5.5.7.3.4 | MAY          |
+| `id-kp-timeStamping`    | 1.3.6.1.5.5.7.3.8 | MAY          |
+| `id-kp-OCSPSigning`     | 1.3.6.1.5.5.7.3.9 | MAY          |
+| `anyExtendedKeyUsage`   | 2.5.29.37.0       | MUST NOT     |
+| Any other value         | -                 | SHOULD NOT   |
+
+##### 7.1.2.8.6 Extended Key Usage - TLS CA
+
+| __Key Purpose__         | __OID__           | __Presence__ |
+| ----                    | ----              | -            |
+| `id-kp-serverAuth`      | 1.3.6.1.5.5.7.3.1 | MUST         |
+| `id-kp-clientAuth`      | 1.3.6.1.5.5.7.3.2 | MAY          |
+| `id-kp-codeSigning`     | 1.3.6.1.5.5.7.3.3 | MUST NOT     |
+| `id-kp-emailProtection` | 1.3.6.1.5.5.7.3.4 | MUST NOT     |
+| `id-kp-timeStamping`    | 1.3.6.1.5.5.7.3.8 | MUST NOT     |
+| `id-kp-OCSPSigning`     | 1.3.6.1.5.5.7.3.9 | MUST NOT     |
+| `anyExtendedKeyUsage`   | 2.5.29.37.0       | MUST NOT     |
```

Comment:
Did you perhaps ignore Section 7.1.2.8 when reading this, which warns readers to read the profile, and that they can't just use these extensions?

I *think* any confusion might have been caused by draft numbering as I was renumbering sections (resulting in three 7.1.2.8.6 instead of three different entries, like it was supposed to be). The links go to the right place, but the numbers don't make that clear.

In this case, the section you're commenting on here is `71268-extended-key-usage---tls-ca`, which is only permitted by 7.1.2.4.1 (Technically Constrained TLS sub-CA) and 7.1.2.5.1 (TLS Subordinate CA).

A cross-certificate (7.1.2.2.2) explicitly references 7.1.2.2.23, which addresses this.

I'm resolving this as WONTFIX/INVALID.